test/steps/openmldb_test.properties

```diff
@@ -1,5 +1,6 @@
-JAVA_SDK_VERSION=0.3.0
-OPENMLDB_SERVER_VERSION=0.3.0
-PYTHON_SDK_VERSION=0.3.0
-BATCH_VERSION=0.3.0-allinone-SNAPSHOT
-DIFF_VERSIONS=0.2.3
+JAVA_SDK_VERSION=0.4.0
+JAVA_NATIVE_VERSION=0.4.0
+OPENMLDB_SERVER_VERSION=0.4.0
+PYTHON_SDK_VERSION=0.4.0
+BATCH_VERSION=0.4.0-allinone-SNAPSHOT
```

Comment:
`openmldb-batch` has only one version. If want to run in different platform,  related `openmldb-native` should be imported

Reply:
Modify the batch version in the next pr